index.bs

```diff
@@ -2263,6 +2556,92 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The browser.setClientWindowState Command #### {#command-browser-setClientWindowState}
+
+The <dfn export for=commands>browser.setClientWindowRect</dfn> command sets the
+dimensions of a [=client window=].
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      browser.SetClientWindowState = (
+        method: "browser.setClientWindowState",
+        params: browser.SetClientWindowStateParameters
+      )
+
+      browser.SetClientWindowStateParameters = {
+        clientWindow: browser.ClientWindow,
+        browser.ClientWindowNamedState // browser.ClientWindowRectState
```

Comment:
Should we also add the option to focus a given window by having a separate property here? Otherwise we could have it similar for navigables as a separate command.

Reply:
I'm inclined to leave this to a followup. You can always focus a specific traversable using `BrowsingContext.activate` and that has the side effect of changing the system focus. A command just to focus the client window without specifying a specific browsing context doesn't have such obvious use cases.

Reply:
As long as we know which window contains which elements I would agree that we should use the `browsingContext.activate` command instead to focus/activate a given window. Lets drop it here.